Silent crash "android.database.sqlite.SQLiteException: too many SQL variables (code 1 SQLITE_ERROR[1]): , while compiling: SELECT"

UUID: 5f2f724e67e3afd6fbdf0d169459a0e2
Build flavor: dev
Version name: v0.1.1
OS info: Android 10, sdk version: 29
Report description: huuh
Logs:
```
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() called, newView==null: false, previousView==null: false, listener==null: true, animationStyle=PUSH
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() previousAnimation onAnimationEndInternal(end==true: true), animationsCount=2, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEndInternal(end==true: true), animationsCount=1, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEnded is null
D/KurobaEx-dev | Chan: vvv App went background vvv
D/KurobaEx-dev | StartActivity: stop
D/KurobaEx-dev | ChanThreadLoader: Cleared timer
D/KurobaEx-dev | BookmarkWatcherController: Calling onBookmarksChanged() app visibility changed (applicationVisibility = Background)
D/KurobaEx-dev | BookmarksManager: persistBookmarks blocking called
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_0 @coroutine#154156]: persist(5)
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_0 @coroutine#154156]: persist() inserted/updated 0 bookmarks, deleted 0 bookmarks
D/KurobaEx-dev |  BookmarksRepository: [arch_disk_io_0 @coroutine#154156]: persist(5) took 1.59ms
D/KurobaEx-dev | BookmarksManager: persistBookmarks blocking finished
D/KurobaEx-dev | HistoryNavigationManager: persistNavigationStack blocking called
D/KurobaEx-dev |  HistoryNavigationRepository: [arch_disk_io_1 @coroutine#154159]: persist(33) took 11.7ms
D/KurobaEx-dev | HistoryNavigationManager: persistNavigationStack blocking finished
D/KurobaEx-dev | BookmarkWatcherController: Switching to background watcher, replaceCurrent=true
D/KurobaEx-dev | BookmarkWatcherController: cancelForegroundBookmarkWatching() called
D/KurobaEx-dev | BookmarkWatcherController: startBackgroundBookmarkWatchingWorkIfNeeded() called
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() called, isCalledFromForeground=false, isUpdatingCurrentlyOpenedThread=false
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=false, isFirstFetch=false) Skipping TD{4chan/a/208035507} because it is a currently opened thread
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=false), Skipping TD{4chan/vg/305330780} because it is not an active bookmark
D/KurobaEx-dev | LastPageNotificationsHelper: showOrUpdateNotifications(3)
D/KurobaEx-dev | LastPageNotificationsHelper: No threads to notify about last page
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: FetchThreadBookmarkInfoUseCase.execute(3)
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://2ch.hk/mobi/res/1947959.json
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://2ch.hk/mobi/res/1939927.json
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://2ch.hk/mobi/res/1952535.json
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{2ch.hk/mobi/1947959}, originalPost=OriginalPost(postNo=1947959, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=NotSticky)
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{2ch.hk/mobi/1939927}, originalPost=OriginalPost(postNo=1939927, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=NotSticky)
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{2ch.hk/mobi/1952535}, originalPost=OriginalPost(postNo=1952535, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=NotSticky)
D/KurobaEx-dev | BookmarkWatcherDelegate: fetchThreadBookmarkInfo stats: total results=3, errorsCount=0, alreadyDeletedCount=0, notFoundOnServerCount=0, badStatusCount=0, successCount=3
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() success, activeBookmarksCount=4
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() took 282ms
D/KurobaEx-dev | ReplyNotificationsHelper: showNotificationForReplies(0)
D/KurobaEx-dev | Chan: ^^^ App went foreground ^^^
D/KurobaEx-dev | StartActivity: start
D/KurobaEx-dev | ChanThreadLoader: Requested thread /a/, 208035507
D/KurobaEx-dev | BookmarkWatcherController: Calling onBookmarksChanged() app visibility changed (applicationVisibility = Foreground)
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() called, isCalledFromForeground=true, isUpdatingCurrentlyOpenedThread=true
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=true) Skipping TD{2ch.hk/mobi/1947959} because it is not a currently opened thread
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=true) Skipping TD{2ch.hk/mobi/1939927} because it is not a currently opened thread
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=true) Skipping TD{4chan/vg/305330780} because it is not a currently opened thread
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=true) Skipping TD{2ch.hk/mobi/1952535} because it is not a currently opened thread
D/KurobaEx-dev | LastPageNotificationsHelper: showOrUpdateNotifications(1)
D/KurobaEx-dev | LastPageNotificationsHelper: Skipping notification for currently opened thread (TD{4chan/a/208035507})
D/KurobaEx-dev | LastPageNotificationsHelper: No threads to notify about last page
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: FetchThreadBookmarkInfoUseCase.execute(1)
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://a.4cdn.org/a/thread/208035507.json
D/KurobaEx-dev | ArchivesManager: getArchiveDescriptor(threadDescriptor=TD{4chan/a/208035507}, forced=false)
D/KurobaEx-dev | ArchivesManager: getBestPossibleArchiveOrNull(threadDescriptor=TD{4chan/a/208035507}, suitableArchivesSize=1)
D/KurobaEx-dev | ArchivesManager: sortedFetchHistoryList[0]: archiveDescriptor=ArchiveDescriptor(archiveDatabaseId='9', name='Archived.Moe', domain='archived.moe'), score=5
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{4chan/a/208035507}, originalPost=OriginalPost(postNo=208035507, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=StickyUnlimited)
D/KurobaEx-dev | BookmarkWatcherDelegate: fetchThreadBookmarkInfo stats: total results=1, errorsCount=0, alreadyDeletedCount=0, notFoundOnServerCount=0, badStatusCount=0, successCount=1
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() success, activeBookmarksCount=4
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() took 189ms
D/KurobaEx-dev | ReplyNotificationsHelper: bookmarksManager.listenForBookmarksChanges(), bookmarkChange=BookmarksUpdated
D/KurobaEx-dev | GetPostsFromArchiveUseCase: Got archive descriptor: ArchiveDescriptor(archiveDatabaseId='9', name='Archived.Moe', domain='archived.moe')
D/KurobaEx-dev |  ArchivesRemoteSource: [DefaultDispatcher-worker-13 @ChanThreadLoaderCoordinator#156802]: fetchThreadFromNetwork(https://archived.moe/_/api/chan/thread/?board=a&num=208035507, 208035507)
D/KurobaEx-dev | BookmarkWatcherController: Switching to foreground watcher
D/KurobaEx-dev | BookmarkWatcherController: cancelBackgroundBookmarkWatching() called
D/KurobaEx-dev | BookmarkWatcherController: bookmarkForegroundWatcher.startWatching() called
D/KurobaEx-dev | BookmarkForegroundWatcher: working == true, calling doWorkAndWaitUntilNext()
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() called, isCalledFromForeground=true, isUpdatingCurrentlyOpenedThread=false
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=false, isFirstFetch=false) Skipping TD{4chan/a/208035507} because it is a currently opened thread
D/KurobaEx-dev | BookmarkWatcherDelegate: (isUpdatingCurrentlyOpenedThread=false), Skipping TD{4chan/vg/305330780} because it is not an active bookmark
D/KurobaEx-dev | LastPageNotificationsHelper: showOrUpdateNotifications(3)
D/KurobaEx-dev | LastPageNotificationsHelper: No threads to notify about last page
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: FetchThreadBookmarkInfoUseCase.execute(3)
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://2ch.hk/mobi/res/1952535.json
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://2ch.hk/mobi/res/1947959.json
D/KurobaEx-dev | FetchThreadBookmarkInfoUseCase: fetchThreadBookmarkInfo() threadJsonEndpoint = https://2ch.hk/mobi/res/1939927.json
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{2ch.hk/mobi/1947959}, originalPost=OriginalPost(postNo=1947959, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=NotSticky)
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{2ch.hk/mobi/1939927}, originalPost=OriginalPost(postNo=1939927, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=NotSticky)
D/KurobaEx-dev | BookmarkWatcherDelegate: FetchResult.Success: descriptor=TD{2ch.hk/mobi/1952535}, originalPost=OriginalPost(postNo=1952535, closed=false, archived=false, isBumpLimit=false, isImageLimit=false, stickyThread=NotSticky)
D/KurobaEx-dev | BookmarkWatcherDelegate: fetchThreadBookmarkInfo stats: total results=3, errorsCount=0, alreadyDeletedCount=0, notFoundOnServerCount=0, badStatusCount=0, successCount=3
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() success, activeBookmarksCount=4
D/KurobaEx-dev | BookmarkWatcherDelegate: BookmarkWatcherDelegate.doWork() took 145ms
D/KurobaEx-dev | BookmarkForegroundWatcher: bookmarkWatcherDelegate.doWork() completed, waiting for 30000ms + 0ms (activeBookmarksCount: 4, total wait time: 30000ms)
D/KurobaEx-dev | BookmarksManager: persistBookmarks async called
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_0 @coroutine#156816]: persist(5)
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_0 @coroutine#156816]: persist() inserted/updated 0 replies
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_0 @coroutine#156816]: persist() inserted/updated 1 bookmarks, deleted 0 bookmarks
D/KurobaEx-dev |  BookmarksRepository: [arch_disk_io_0 @coroutine#156816]: persist(5) took 3.07ms
D/KurobaEx-dev | BookmarksManager: persistBookmarks async finished
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() called, newView==null: false, previousView==null: false, listener==null: true, animationStyle=POP
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() previousAnimation onAnimationEndInternal(end==true: true), animationsCount=2, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEndInternal(end==true: true), animationsCount=1, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEnded is null
D/KurobaEx-dev | ReplyNotificationsHelper: showNotificationForReplies(0)
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() called, newView==null: false, previousView==null: false, listener==null: true, animationStyle=PUSH
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() previousAnimation onAnimationEndInternal(end==true: true), animationsCount=2, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEndInternal(end==true: true), animationsCount=1, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEnded is null
D/KurobaEx-dev | GetPostsFromArchiveUseCase: Successfully fetched 2630 posts from archive ArchiveDescriptor(archiveDatabaseId='9', name='Archived.Moe', domain='archived.moe')
E/KurobaEx-dev | NormalPostLoader: Error while trying to get posts from archive
E/KurobaEx-dev | NormalPostLoader: android.database.sqlite.SQLiteException: too many SQL variables (code 1 SQLITE_ERROR[1]): , while compiling: SELECT 
E/KurobaEx-dev | NormalPostLoader:         	  cp_image.post_image_id as post_image_id,
E/KurobaEx-dev | NormalPostLoader:         	  cp_id.post_id as post_id, 
E/KurobaEx-dev | NormalPostLoader:         	  cp_id.post_no as post_no, 
E/KurobaEx-dev | NormalPostLoader:         	  cp_id.owner_archive_id as archive_id
E/KurobaEx-dev | NormalPostLoader:         FROM chan_post_id cp_id
E/KurobaEx-dev | NormalPostLoader:         LEFT OUTER JOIN chan_post_image cp_image
E/KurobaEx-dev | NormalPostLoader:             ON cp_image.owner_post_id = cp_id.post_id
E/KurobaEx-dev | NormalPostLoader:         WHERE 
E/KurobaEx-dev | NormalPostLoader:             cp_id.owner_thread_id = ?
E/KurobaEx-dev | NormalPostLoader:         AND
E/KurobaEx-dev | NormalPostLoader:             cp_id.owner_archive_id IN (?,?)
E/KurobaEx-dev | NormalPostLoader:         AND 
E/KurobaEx-dev | NormalPostLoader:             cp_id.post_no IN (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
E/KurobaEx-dev | NormalPostLoader: ,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1372)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:811)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:590)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:62)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteQuery.<init>(SQLiteQuery.java:37)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:46)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1967)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1942)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.sqlite.db.framework.FrameworkSQLiteDatabase.query(FrameworkSQLiteDatabase.java:161)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.RoomDatabase.query(RoomDatabase.java:328)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.util.DBUtil.query(DBUtil.java:83)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao_Impl$16.call(ChanPostDao_Impl.java:514)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao_Impl$16.call(ChanPostDao_Impl.java:511)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.CoroutinesRoom$Companion.execute(CoroutinesRoom.kt:46)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.CoroutinesRoom.execute(Unknown Source:2)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao_Impl.selectManyGrouped(ChanPostDao_Impl.java:511)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao.selectMany(ChanPostDao.kt:77)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.source.local.ChanPostLocalSource.getThreadPosts(ChanPostLocalSource.kt:394)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.repository.ChanPostRepository$getThreadPosts$3$1.invokeSuspend(ChanPostRepository.kt:236)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.repository.ChanPostRepository$getThreadPosts$3$1.invoke(Unknown Source:10)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.RoomDatabaseKt$withTransaction$2.invokeSuspend(RoomDatabase.kt:58)
E/KurobaEx-dev | NormalPostLoader: 	at (Coroutine boundary.()
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.RoomDatabaseKt.withTransaction(RoomDatabase.kt:51)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.repository.AbstractRepository.tryWithTransaction(AbstractRepository.kt:16)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.repository.ChanPostRepository$getThreadPosts$3.invokeSuspend(ChanPostRepository.kt:226)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.common.KotlinExtensionsKt$myAsync$2$1.invokeSuspend(KotlinExtensions.kt:73)
E/KurobaEx-dev | NormalPostLoader: Caused by: android.database.sqlite.SQLiteException: too many SQL variables (code 1 SQLITE_ERROR[1]): , while compiling: SELECT 
E/KurobaEx-dev | NormalPostLoader:         	  cp_image.post_image_id as post_image_id,
E/KurobaEx-dev | NormalPostLoader:         	  cp_id.post_id as post_id, 
E/KurobaEx-dev | NormalPostLoader:         	  cp_id.post_no as post_no, 
E/KurobaEx-dev | NormalPostLoader:         	  cp_id.owner_archive_id as archive_id
E/KurobaEx-dev | NormalPostLoader:         FROM chan_post_id cp_id
E/KurobaEx-dev | NormalPostLoader:         LEFT OUTER JOIN chan_post_image cp_image
E/KurobaEx-dev | NormalPostLoader:             ON cp_image.owner_post_id = cp_id.post_id
E/KurobaEx-dev | NormalPostLoader:         WHERE 
E/KurobaEx-dev | NormalPostLoader:             cp_id.owner_thread_id = ?
E/KurobaEx-dev | NormalPostLoader:         AND
E/KurobaEx-dev | NormalPostLoader:             cp_id.owner_archive_id IN (?,?)
E/KurobaEx-dev | NormalPostLoader:         AND 
E/KurobaEx-dev | NormalPostLoader:             cp_id.post_no IN (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
E/KurobaEx-dev | NormalPostLoader: ,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:1372)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:811)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:590)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:62)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteQuery.<init>(SQLiteQuery.java:37)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:46)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1967)
E/KurobaEx-dev | NormalPostLoader: 	at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1942)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.sqlite.db.framework.FrameworkSQLiteDatabase.query(FrameworkSQLiteDatabase.java:161)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.RoomDatabase.query(RoomDatabase.java:328)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.util.DBUtil.query(DBUtil.java:83)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao_Impl$16.call(ChanPostDao_Impl.java:514)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao_Impl$16.call(ChanPostDao_Impl.java:511)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.CoroutinesRoom$Companion.execute(CoroutinesRoom.kt:46)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.CoroutinesRoom.execute(Unknown Source:2)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao_Impl.selectManyGrouped(ChanPostDao_Impl.java:511)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.dao.ChanPostDao.selectMany(ChanPostDao.kt:77)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.source.local.ChanPostLocalSource.getThreadPosts(ChanPostLocalSource.kt:394)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.repository.ChanPostRepository$getThreadPosts$3$1.invokeSuspend(ChanPostRepository.kt:236)
E/KurobaEx-dev | NormalPostLoader: 	at com.github.adamantcheese.model.repository.ChanPostRepository$getThreadPosts$3$1.invoke(Unknown Source:10)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.RoomDatabaseKt$withTransaction$2.invokeSuspend(RoomDatabase.kt:58)
E/KurobaEx-dev | NormalPostLoader: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:84)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
E/KurobaEx-dev | NormalPostLoader: 	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.RoomDatabaseKt$acquireTransactionThread$$inlined$suspendCancellableCoroutine$lambda$2.run(RoomDatabase.kt:121)
E/KurobaEx-dev | NormalPostLoader: 	at androidx.room.TransactionExecutor$1.run(TransactionExecutor.java:45)
E/KurobaEx-dev | NormalPostLoader: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/KurobaEx-dev | NormalPostLoader: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/KurobaEx-dev | NormalPostLoader: 	at java.lang.Thread.run(Thread.java:919)
D/KurobaEx-dev | ParsePostsUseCase: parseNewPostsPosts(chanDescriptor=TD{4chan/a/208035507}, postsToParseSize=2619, maxCount=2147483647)
D/KurobaEx-dev | StoreNewPostsUseCase: storePosts(postsCount=2619, isCatalog=false)
D/KurobaEx-dev |  ChanPostRepository: [DefaultDispatcher-worker-1 @ChanThreadLoaderCoordinator#154172]: getThreadPosts(descriptor=TD{4chan/a/208035507}, archiveIds=[9, 0], maxCount=2147483647)
D/KurobaEx-dev | NormalPostLoader: ChanReaderRequest.readJson() stats: url = https://a.4cdn.org/a/thread/208035507.json.
D/KurobaEx-dev | NormalPostLoader: Store new posts took 441ms (stored 2618 posts).
D/KurobaEx-dev | NormalPostLoader: Reload posts took 187ms, (reloaded 2621 posts, from cache: 0).
D/KurobaEx-dev | NormalPostLoader: Parse posts took = 616ms, (parsed 2619 posts).
D/KurobaEx-dev | NormalPostLoader: Archive fetch took 5.60s, (fetched 0 deleted posts).
D/KurobaEx-dev | NormalPostLoader: Total in-memory cached posts count = (0/26214).
D/KurobaEx-dev | ThreadPresenter: onChanLoaderData() called
D/KurobaEx-dev | ChanThreadLoader: Scheduled reload in 15s
D/KurobaEx-dev | ImageLoaderV2: loadFromNetwork(url=https://i.4cdn.org/a/1598871268387s.jpg, width=213, height=213)
D/KurobaEx-dev |  ChanThreadViewableInfoLocalSource: [arch_disk_io_2 @coroutine#159472]: Updating ChanThreadViewableInfo for TD{4chan/a/208035507}
D/KurobaEx-dev | BookmarksManager: delayedBookmarksChanged(BookmarksUpdated)
D/KurobaEx-dev | ReplyNotificationsHelper: bookmarksManager.listenForBookmarksChanges(), bookmarkChange=BookmarksUpdated
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() called, newView==null: false, previousView==null: false, listener==null: true, animationStyle=POP
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() previousAnimation onAnimationEndInternal(end==true: true), animationsCount=2, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEndInternal(end==true: true), animationsCount=1, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEnded is null
D/KurobaEx-dev | BookmarksManager: persistBookmarks async called
D/KurobaEx-dev | ReplyNotificationsHelper: showNotificationForReplies(0)
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_3 @coroutine#159476]: persist(5)
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_3 @coroutine#159476]: persist() inserted/updated 0 replies
D/KurobaEx-dev |  ThreadBookmarkLocalSource: [arch_disk_io_3 @coroutine#159476]: persist() inserted/updated 1 bookmarks, deleted 0 bookmarks
D/KurobaEx-dev |  BookmarksRepository: [arch_disk_io_3 @coroutine#159476]: persist(5) took 5.12ms
D/KurobaEx-dev | BookmarksManager: persistBookmarks async finished
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() called, newView==null: false, previousView==null: false, listener==null: true, animationStyle=PUSH
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() previousAnimation onAnimationEndInternal(end==true: true), animationsCount=2, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEndInternal(end==true: true), animationsCount=1, listenerCalled=false
D/KurobaEx-dev | ToolbarContainer: setPushPopAnimation() newAnimation onAnimationEnded is null
D/KurobaEx-dev | AndroidUtils: Build Signature: DC5195CC40E42B95267D500B6E93E46EC51028C67BDD3D09BBB9C208BF20C8FE
D/KurobaEx-dev | AndroidUtils: Build Signature: DC5195CC40E42B95267D500B6E93E46EC51028C67BDD3D09BBB9C208BF20C8FE
D/KurobaEx-dev | AndroidUtils: waitForLayout requested a re-layout by returning false
D/KurobaEx-dev | AndroidUtils: Build Signature: DC5195CC40E42B95267D500B6E93E46EC51028C67BDD3D09BBB9C208BF20C8FE
D/KurobaEx-dev | AndroidUtils: Build Signature: DC5195CC40E42B95267D500B6E93E46EC51028C67BDD3D09BBB9C208BF20C8FE
```

--- a/Kuroba/core-model/src/main/java/com/github/adamantcheese/model/dao/ChanPostDao.kt
+++ b/Kuroba/core-model/src/main/java/com/github/adamantcheese/model/dao/ChanPostDao.kt
@@ -74,11 +74,10 @@ abstract class ChanPostDao {
     ownerArchiveIds: Collection<Long>,
     postNos: Collection<Long>
   ): List<ChanPostFull> {
-    val groupedPostIdPostNoDto = selectManyGrouped(
-      ownerThreadId,
-      ownerArchiveIds,
-      postNos
-    )
+    val groupedPostIdPostNoDto = postNos
+      .chunked(KurobaDatabase.SQLITE_IN_OPERATOR_MAX_BATCH_SIZE)
+      .flatMap { chunk -> selectManyGrouped(ownerThreadId, ownerArchiveIds, chunk)
+    }
 
     return retainMostValuablePosts(groupedPostIdPostNoDto)
       .chunked(KurobaDatabase.SQLITE_IN_OPERATOR_MAX_BATCH_SIZE)

